app/packages/acter_trigger_auto_complete/lib/src/acter_trigger_autocomplete.dart

```diff
@@ -377,13 +377,13 @@ class MultiTriggerAutocompleteState extends State<MultiTriggerAutocomplete> {
 
   @override
   void dispose() {
-    _textEditingController.removeListener(_onChangedField);
-    if (widget.textEditingController == null) {
+    if (!mounted) {
```

Comment:
@gnunicorn, I think we don't want dispose off controller here at all. As doing interaction on chat screen causes this to dispose which I'm not sure why...